.github/workflows/create-quantic-package.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           node-version: 14.x
       - name: Check Out Repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Is the bump required for a new feature/improvement, or just for "maintenance" ? Just curious 🧐

Reply:
Just maintenance, everything github action v3 is available